CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Added
+- Added `GetPendingSalesOrderData` in `BusinessCore()` source.
```

Comment:
Is the `GetPendingSalesOrderData` some sort of endpoint?

Reply:
Yes, added this to the changelog to be clear